src/scripts/main.js

```diff
@@ -1,87 +1,89 @@
-const body = document.querySelector('body');
-
-function createNotification(message, type) {
-  const notification = document.createElement('div');
-
-  notification.setAttribute('data-qa', 'notification');
-  notification.className = type;
-  notification.innerText = message;
-  body.insertAdjacentElement('afterbegin', notification);
-}
-
-function firstPromise() {
-  return new Promise((resolve, reject) => {
-    let isClicked = false;
-
-    const timeOut = setTimeout(() => {
-      createNotification('First promise was rejected', 'error');
-      // eslint-disable-next-line prefer-promise-reject-errors
-      reject('First promise was rejected');
-      isClicked = true;
-    }, 3000);
-
-    document.addEventListener('click', (e) => {
-      if (e.button === 0 && !isClicked) {
-        isClicked = true;
-        createNotification('First promise was resolved', 'success');
-        clearTimeout(timeOut);
-        resolve('First promise was resolved');
-      }
-    });
-  });
-}
+document.addEventListener('contextmenu', (eve) => {
+  eve.preventDefault();
+});
 
-function secondPromise() {
-  return new Promise((resolve) => {
-    let isClicked = false;
+const firstPromise = new Promise((resolve, reject) => {
+  const rejectTimeOut = setTimeout(() => {
+    const div = document.createElement('div');
 
-    document.addEventListener('mousedown', (e) => {
-      if ((e.button === 0 || e.button === 2) && !isClicked) {
-        isClicked = true;
-        createNotification('Second promise was resolved', 'success');
-        resolve('Second promise was resolved');
-      }
-    });
+    div.setAttribute('data-qa', 'notification');
+    div.classList.add('error');
+    div.textContent = 'First promise was rejected';
```

Comment:
Creating a div element and adding attributes (data-qa and class) is repeated multiple times in the code. This could be refactored into a reusable function to avoid repetition.

Don't know why did you changed that 😭